src/Microsoft.VisualStudio.ProjectSystem.CSharp/ProjectSystem/Rules/CSharpProjectProperties.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.ComponentModel.Composition;
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.VisualStudio.ProjectSystem.Properties;
+
+namespace Microsoft.VisualStudio.ProjectSystem
+{
+    /// <summary>
+    ///     Represent the C# project properties.
+    /// </summary>
+    [Export]
+    [ExcludeFromCodeCoverage]
+    internal partial class CSharpProjectProperties : ProjectProperties
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ProjectProperties"/> class.
+        /// </summary>
+        [ImportingConstructor]
+        public CSharpProjectProperties([Import] ConfiguredProject configuredProject)
```

Comment:
If we can think of a better name here that'll be good. ProjectProperties is very confusing when you consider that CPS's IProjectPropertiesProvider is something else.